allofplos/samples/corpus_analysis.py

```diff
@@ -351,7 +351,7 @@ def get_article_metadata(article_file, size='small'):
     """
     article = Article.from_filename(article_file)
     doi = article.doi
-    filename = os.path.basename(article.filename).rstrip('.xml')
+    filename = os.path.basename(article.filepath).rstrip('.xml')
```

Comment:
Is there a reason the filename shouldn't include the extension? It technically is part of the filename.